### PR TITLE
WPP-133 Rework Database to Allow for Profile Pictures - Backend

### PR DESCRIPTION
**Pull request by Blackkirby72**
## For Pull Requesters/Reviewers:
All of the following should be true before making a pull request and should be inspected by reviewers:

### User Story
- [x] Acceptance criteria are adhered to and completely finished
- [x] All subtasks on Jira are completed

### Unit Testing
- [x] Unit tests are finished and passing
- [x] Code coverage is >80%

### Documentation/Style
- [x] JSDocs are written for methods and components
- [x] Unclear code has been documented
- [x] Style standards are adhered to

## For Reviewers:
The following should be additionally checked by reviewers:

- [x] Code runs locally
- [ ] Visually inspect tests; they should not simply pass, but rather they should be doing something and testing acceptance criteria
- [x] See if "jank" code could be rewritten

## Description
Updated the database to contain columns for `profile_picture`, `profile_picture_top`, and `profile_picture_left` which allows the front end to crop as it sees fit. 

It may seem a little iffy, but the way it's written allows us to just update our database when starting the project.

### CODE REVIEW

**Comment by PistolRcks** [comment]
Please pull from master first then resolve your merge conflicts.

**Review by lmschaff — APPROVED** [review]
looks good!

File: server/db.js [hunk 1]
```diff
@@ -71,6 +73,25 @@ function initDatabase(fp, puzzles) {
   return db;
 }
 
+const addColumns = (db) => {
```
Comment (PistolRcks) [inline server/db.js:81]: Please add JSDocs as per the DoD.

**Review by PistolRcks — CHANGES_REQUESTED** [review]

**Review by connormckellips — APPROVED** [review]
Looks good to me. Syntax errors are the worst lol

**Review by PistolRcks — APPROVED** [review]

**Pull request merged**